tests/bsim/bluetooth/host/att/read_fill_buf/test_scripts/run_tests.sh

```diff
@@ -8,17 +8,17 @@ source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
 verbosity_level=2
 simulation_id="read_fill_buf"
-test_exe_d0="${BSIM_OUT_PATH}/bin/bs_${BOARD_TS}_$(guess_test_long_name)_client_prj_conf"
-test_exe_d1="${BSIM_OUT_PATH}/bin/bs_${BOARD_TS}_$(guess_test_long_name)_server_prj_conf"
+test_exe_d0="./bs_${BOARD_TS}_$(guess_test_long_name)_client_prj_conf"
+test_exe_d1="./bs_${BOARD_TS}_$(guess_test_long_name)_server_prj_conf"
 
 cd ${BSIM_OUT_PATH}/bin
```

Comment:
This works, but somehow I would feel better if we had the `cd` before assigning the relative paths, even though it doesn't really matter as long as the `cd` is before the `Execute`

Reply:
Relative pathing is used in most of the test_scripts, so I'd rather this be done somewhere else!